packages/flutter/lib/src/gestures/events.dart

```diff
@@ -86,6 +130,44 @@ int nthMouseButton(int number) => (kPrimaryMouseButton << (number - 1)) & kMaxUn
 /// for some stylus buttons.
 int nthStylusButton(int number) => (kPrimaryStylusButton << (number - 1)) & kMaxUnsignedSMI;
 
+/// Returns the button of `buttons` with the smallest integer.
+///
+/// A `buttons` is a bitfield where each set bit represents a button. This
```

Comment:
```suggestion
/// The [buttons] argument is a bitfield where each set bit represents a button. This
```

Reply:
Please use \`\` for arguments. See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements